minitask/new_import_data_v2_multip.py

```diff
@@ -0,0 +1,140 @@
+
```

Comment:
**[Required]** Why does this duplicate file exist? It's not needed.

Reply:
this is the one we use right